checker_c#/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+WORKDIR /src
+
+# Fetch dependencies
+COPY N0t3b00kChecker.sln .
+COPY N0t3b00kChecker/N0t3b00kChecker.csproj N0t3b00kChecker/N0t3b00kChecker.csproj
+RUN dotnet restore
+
+# Publish
+COPY . .
+RUN dotnet publish -c Release --self-contained true -o /build
+
+# Copy to runtime container
+FROM ghcr.io/enowars/enochecker-dotnet:nightly

```

Comment:
is it best practice to use the nightly version?

Reply:
I am torn between nightly for faster CI fails and stable for a better example 🤔

Reply:
can't we just (ab)use gamemaster for testing the nightly with CI?